src/sage/rings/polynomial/polynomial_zz_pex.pyx

```diff
@@ -223,13 +223,13 @@ cdef class Polynomial_ZZ_pEX(Polynomial_template):
         return r
 
     def __call__(self, *x, **kwds):
-        """
+        r"""
         Evaluate polynomial at `a`.
 
         EXAMPLES::
 
-            sage: K.<u>=GF(next_prime(2**60)**3)
-            sage: R.<x> = PolynomialRing(K,implementation='NTL')
+            sage: K.<u> = GF(next_prime(2**60)**3)
+            sage: R.<x> = PolynomialRing(K, implementation='NTL')
             sage: P = (x-u)*(x+u+1)
```

Comment:
`sage.rings.finite_rings` and `sage.libs.ntl`?

Reply:
The `sage.libs.ntl` will be redundant because this file has a compile-time dependency on NTL anyway, but I've added both tags for the file.